packages/wonder-blocks-clickable/src/components/__tests__/clickable-behavior.test.js

```diff
@@ -297,16 +297,40 @@ describe("ClickableBehavior", () => {
             </ClickableBehavior>,
         );
 
+        // Assert
+        const button = screen.getByTestId("test-button-1");
+        expect(button).not.toHaveAttribute("tabIndex");
+    });
+
+    test("should have the tabIndex that is passed in", () => {
+        // Arrange
+        // Act
+        render(
+            <ClickableBehavior
+                disabled={false}
+                onClick={(e) => {}}
+                tabIndex={0}
+            >
+                {(state, childrenProps) => {
+                    return (
+                        <button data-test-id="test-button-1" {...childrenProps}>
+                            Label
+                        </button>
+                    );
+                }}
+            </ClickableBehavior>,
+        );
+
```

Comment:
**suggestion:** I think it would be useful to include another test where we include a non-interactive element and pass in `tabIndex=0` then verify if it can be focused to ensure that the tabIndex is set correctly (something similar to the `WithTabIndex` story might work).